### DISCUSSION TITLE
breaking the frame

### DISCUSSION THREAD

**Discussion by alsfangelo** [splinetool/react-spline#97] [Q&A]
![Screenshot_20221129_011229_Chrome.jpg](https://user-images.githubusercontent.com/98702765/204438369-fdb7c6db-ff08-456a-8388-9bc6707dd2b2.jpg)
After some clicks on the iconTheme (moon) a little face appears in the upper corner of the keyboard, this blue background appears (it does not exist before breaking) and it does not work anymore

**Comment by marcofugaro** [comment]
Can you share the link?

**Comment by alsfangelo** [comment]
> Can you share the link?
> 

https://angelollima-homepage.vercel.app/

this case only happens on the cell phone (keep changing themes until the keyboard breaks)

**Comment by marcofugaro** [comment]
Looks like you're using multiple canvases, you can't instantiate multiple `<Spline` at once, it's very resource intensive, even if they are hidden with `display: none;`.

You should use React's render logic to hide/show them.

**Reply by alsfangelo** [reply]
I did that and it keeps bugging.

**Reply by marcofugaro** [reply]
@angelollima how do I reproduce the error?

**Reply by alsfangelo** [reply]
@marcofugaro on your mobile phone, keeps clicking on the moon/sun icons in order to change the theme, after about 6 clicks you will see that the keyboard background will change
